packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2405,3 +2406,45 @@ const waitForSeq = async (
     };
     deltaManager.on("op", handleOp);
 });
+
+export interface ISummarizerRequestOptions {
+    cache: boolean,
+    reconnect: boolean,
+    summarizingClient: boolean,
+    url: string,
+}
+
+/**
+ * Requests a Summarizer.
+ * @param loader - the loader that resolves the request
+ * @param lastSequenceNumber - the last sequence number (e.g., from DeltaManager)
+ * @param cache - use cache to retrieve summarizer
+ * @param summarizingClient - is summarizer client
+ * @param reconnect - can reconnect on connection loss
+ */
+export async function requestSummarizer(
```

Comment:
Can this function be somewhere else (i.e. not in containerRuntime.ts)? Closer to summarizer code?

Reply:
i think all the summarizer code is currently co-located in the container runtime package. i would love to see this all put in and independent package like GC does

Reply:
Moved it into the summarizer code.